lib/y2snapper_common.pm

```diff
@@ -65,7 +65,7 @@ It allows to have more control over diffs amongs snapshots.
 
 =cut
 sub y2snapper_adding_new_snapper_conf {
-    assert_script_run("btrfs subvolume create /test");
+    assert_script_run("btrfs subvolume create /test") if (script_run "ls /test");
```

Comment:
I think this condition it would be nice to be documented. But take a look first to the other comments. Because if you move the `x11/yast2_snapper` to some test that makes sense* you wont need this.

Reply:
The condition is needed in case some other module created /test before y2snapper_adding_new_snapper_conf is called. If /test is already created, then "btrfs subvolume create /test" will fail . By "documented" you mean that I should add a comment?

Reply:
basically yes. i mean commented. but consider the other solution also.